projects/postgresql.org/package.yml

```diff
@@ -0,0 +1,97 @@
+distributable:
+   url: https://github.com/postgres/postgres/archive/refs/tags/REL_{{version.major}}_{{version.minor}}.tar.gz
+
+   strip-components: 1
+versions:
+  - 15.2 # need to hardcode this version, because no easy way to get the tag yet 
+
+dependencies:
+  openssl.org: ^1.0.1
+  gnu.org/readline: '*'
+  zlib.net: '*'
+  lz4.org: '*'
+  gnome.org/libxml2: '*'
+  gnome.org/libxslt: '*'
+
+build:
+  dependencies:
+    tea.xyz/gx/make: '*'
+    freedesktop.org/pkg-config: '*'
+    darwin:
+      tea.xyz/gx/cc: c99
+    linux:
+      gnu.org/gcc: '*'
+      gnu.org/bison: '*'
+      github.com/westes/flex: ^2.5.31
+      perl.org: '*'
+  script: |
+    ./configure $ARGS
+    make --jobs {{ hw.concurrency }}
+    make install
+  env:
+    ARGS:
+      - --prefix={{prefix}}
+      - --with-ssl=openssl
+      - --with-lz4
+      - --with-libxml
+      - --with-libxslt
+
+provides:
+  - bin/clusterdb
+  - bin/createdb
+  - bin/dropdb
+  - bin/dropuser
+  - bin/ecpg
+  - bin/initdb
+  - bin/pg_amcheck
+  - bin/pg_archivecleanup
+  - bin/pg_basebackup
+  - bin/pg_config
+  - bin/pg_controldata
+  - bin/pg_ctl
+  - bin/pg_dump
+  - bin/pg_dumpall
+  - bin/pg_isready
+  - bin/pg_receivewal
+  - bin/pg_recvlogical
+  - bin/pg_resetwal
+  - bin/pg_restore
+  - bin/pg_rewind
+  - bin/pg_test_fsync
+  - bin/pg_test_timing
+  - bin/pg_upgrade
+  - bin/pg_verifybackup
+  - bin/pg_waldump
+  - bin/pgbench
+  - bin/postgres
+  - bin/psql
+  - bin/reindexdb
+  - bin/vacuumdb
+
+test:
+  script: |
+    mkdir -p ./data
+
+    if test "{{ hw.platform }}" = "linux"; then
+      adduser postgres --disabled-password --gecos ""
+      chown -R postgres:postgres ./data
+      PASSWD=$(date | md5sum | cut -c1-8)
+      echo "postgres:$PASSWD" | chpasswd
+      su postgres -c 'initdb -D ./data'
```

Comment:
hmm the ci doesn't like this one either... in local docker it works. and `su nobody -c 'initdb -D ./data'` doesn't work. maybe it works in the ci?

Reply:
I think you want `sudo -u nobody initdb -D ./data`. The runners have pw-less sudo, much like GHA runners.